articles/spring-cloud/spring-cloud-quickstart-setup-config-server.md

```diff
@@ -66,6 +66,19 @@ Set up your config-server with the location of the git repository for the projec
 az spring-cloud config-server git set -n <service instance name> --uri https://github.com/Azure-Samples/piggymetrics-config
 ```
 
+## Troubleshooting of Azure Spring Cloud config server
+
+The following procedure tells us how to troubleshoot config server settings.
+
+1. Go to the service **Overview** page and select **Logs**. Select **Queries** and **Show the application logs which contain the "error" or "exception" terms"**. Click **Run**. If you find the error **java.lang.illegalStateException** in logs, which indicates that spring cloud service cannot locate properties from config server.
```

Comment:
If you find the error **java.lang.illegalStateException** in logs, this indicates that spring cloud service cannot locate properties from config server.

Reply:
one word to change: which -> this